release/platform/platform.go

```diff
@@ -353,4 +353,12 @@ var platforms = []Platform{
 		Repos:     []string{RepoOrg, RepoEnterprise},
 		BuildTags: []string{"ssl", "sasl", "gssapi", "failpoints"},
 	},
+	{
+		Name:      "ubuntu1604",
```

Comment:
I know zseries distro would be deprecated for ubuntu systems according to TOOLS-2866. However, this platform is still used for `z Race Detector test`. All the tests in that variant failed after TOOLS-2866. I tried to change the distro to RHEL 8 zseries, but there are other dependencies not working there. Maybe we should create another ticket to fix this variant?

Reply:
Yeah, sounds good!

Reply:
Sounds good to me as well. @huan-Mongo can you link the ticket here once you've filed it?

Reply:
TOOLS-2922 is created.